doc/Comments.xml

```diff
@@ -157,14 +159,25 @@ can be used to set the label of the chapter to <C>Chapter_</C><A>label</A>.
 
 Additionally, the chapter will be stored as <C>_Chapter_</C><A>label</A><C>.xml</C>.
 
+The <C>@ChapterTitle</C> <A>title</A> command
+can be used to set a heading for the chapter that is different from <A>name</A>.
 
+If you use all three commands, i.e.,
+<Listing><![CDATA[
+#! @Chapter name
+#! @ChapterLabel label
+#! @ChapterTitle title
+]]></Listing>
+<C>title</C> is used for the headline, <C>label</C> for cross-referencing, and <C>name</C>
+for setting the same chapter as active chapter again.
```

Comment:
So, if I only use `@Chapter` and `@ChapterTitle`, then what will the chapter label be?

Reply:
No, that's fine. What I was getting at is: So essentially `@Chapter name` is equivalent to `@ChapterLabel name` plus `@ChapterTitle name` (of course it isn't quite the same, because you cannot omit it). 
If that's true, one would never want/need to use all three together. Also, the following two would be equivalent:
```
@Chapter label
@ChapterTitle title
```
vs
```
@Chapter title
@ChapterLabel label
```
Is that correct? If not, in which way do they differ?

If it *is* correct, then I wonder why we need `@ChapterTitle`...

Reply:
They differ in a very particular point:
If you want to have content for the same chapter in different files, you have to reopen the chapter by the name you give it, i.e., in the first example one has to reopen the chapter with
```
@Chapter label
```
and in the second one with
```
@Chapter title
```
So, there actually is a reason to use all three of them together, I think.

Reply:
The examples will by the way appear the same in the manual, but will for example produce different filenames.

Reply:
I find this quite confusing -- why would one ever want to have three different values for these? (Not everything that *can* be done should be done).

However, since Chapter and ChapterLabel exists, we gotta support them for now... Let's just hope nobody will be caught by surprise by this ...